src/components/sendTx/SendTxModal.tsx

```diff
@@ -194,6 +197,45 @@ function SendTxModal({ isOpen, onClose }: SendTxModalProps): JSX.Element {
     isVaultAccount
   );
 
+  const insufficientFunds = O.mapWithDefault(accountState, '', (acc) => {
+    if (!estimatedGas) {
+      return 'Waiting for gas estimation...';
+    }
+
+    const tx = getValues();
+    const fee = estimatedGas * BigInt(tx.gasPrice);
+    if (BigInt(acc.projected.balance) < fee) {
+      return 'You have insufficient funds to pay for gas';
+    }
```

Comment:
Was your intention to display the "insufficient funds for gas" in case the balance is enough to cover the tx amount but the gas fee exceeds the available balance? 
Cause now these lines work only in case the gas is the only thing to pay (spawn), but in the regular tx only the "insufficient funds to send this amount" is displayed.
If it's intended to be like that then it's ok, IMHO no need to overcomplicate these alerts, it's just that the PR description left me a bit hesitating about the intent

Reply:
Got the same after some playing with numbers. Initially it worked.

Reply:
> Was your intention to display the "insufficient funds for gas" in case the balance is enough to cover the tx amount but the gas fee exceeds the available balance?

Well, it will be shown for any transaction, when the balance is not enough to pay for a gas. Including the spend tx.
For example, try to send `0 SMH` with `10000 Smidge / unit` of gas price from the same test account (with 0.001 SMH), and you'll get this error message.

However, for spend tx there is additional condition, that checks that balance is equal or greater than `amount + gas`. And if it is not — then it will show "insufficient funds to send this amount". Basically it means that you have enough to pay for gas, but you should send less :)